src/Screens/Footer/Footer.js

```diff
@@ -78,30 +76,44 @@ const Footer = props => {
           width: dgl * 0.3,
           height: dgl * 0.9,
           alignSelf: 'center',
-          backgroundColor: 'cyan',
         }}
       />
     );
   };
+  if (FooterVisibility) {
+    return (
+      <>
+        <ActionSheetView
+          ref={actionSheetRef}
+          navigation={navigation}
+          onPress={() => handleActionDown()}
+        />
 
-  return (
-    <>
-      <ActionSheetView
-        ref={actionSheetRef}
-        navigation={navigation}
-        onPress={() => handleActionDown()}
-      />
-      {FooterVisibility ? (
         <View style={[styles.footer]}>
           {item.map((item, idx) => {
             return <ItemRender key={idx} value={item} />;
           })}
         </View>
-      ) : (
-        <></>
-      )}
-    </>
-  );
+      </>
+    );
+  }
+  return;
+  // return (
+  //   <>
+  //     <ActionSheetView
+  //       ref={actionSheetRef}
+  //       navigation={navigation}
+  //       onPress={() => handleActionDown()}
+  //     />
+  //     {FooterVisibility && (
+  //       <View style={[styles.footer]}>
+  //         {item.map((item, idx) => {
+  //           return <ItemRender key={idx} value={item} />;
+  //         })}
+  //       </View>
+  //     )}
+  //   </>
+  // );
 };
 
```

Comment:
Check these comments as well. If not needed remove it. Make sure to remove all the comments before committing changes.

Reply:
Removed the comments from screens